tests/pytests/test_rdb_compatibility.py

```diff
@@ -68,12 +65,9 @@ def testRDBCompatibility_vecsim():
     rdbs = ['redisearch_2.4.14_with_vecsim.rdb',
             'redisearch_2.6.9_with_vecsim.rdb']
     algorithms = ['FLAT', 'HNSW']
-    if not downloadFiles(rdbs):
-        if CI:
-            env.assertTrue(False)  ## we could not download rdbs and we are running on CI, let fail the test
-        else:
-            env.skip()
-            return
+    res = downloadFiles(env, rdbs)
+    if not res:
+        return
```

Comment:
```suggestion
    if not downloadFiles(env, RDBS):
        return
```

Reply:
😓 
Wanted to use it for logging or something